src/Components/Profile/MyPosts/Post/post.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import s from './post.module.css';
+
+import logo from "./post.jpg";
+
+const Post = (props) => {
+    return (
+        <div className={s.item}>
+            <img src={logo} alt="logo" /> {props.message}
+           <div>  <span>like</span></div>
```

Comment:
I think there's a problem with indentation - don't forget to press "Shift + Alt + F".

Reply:
i added eslint config and prettier. And now i can use this combination